### ISSUE TITLE
Update to 3.5.0 error: Upgrading entry elements SEO data failed

### ISSUE THREAD

**Issue by ghost** [ethercreative/seo#172]
After updating to 3.5.0 I get the following error (from storage/logs/queue.log):

```
2019-01-21 14:24:49 [-][1][-][info][craft\queue\QueueLogBehavior::beforeExec]  [11436] Upgrading entry elements SEO data (attempt: 1) - Started
2019-01-21 14:24:49 [-][1][-][error][craft\queue\QueueLogBehavior::afterError]  [11436] Upgrading entry elements SEO data (attempt: 1) - Error (time: 0.036s): Setting read-only property: ether\seo\models\data\SeoData::title
```

Am I doing something wrong? Can't edit any entries as I get a server error.

**Comment by shgzon** [comment]
I've got the same problem. It looks like the SEO title and description settings from the SEO entry aren't processed to the Dynamic version.

Despite the fact the update is telling me all old entries get updated...

Please help us with this guys :)

**Comment by ghost** [comment]
Yeah seems like it is because of the update to the dynamic entry title thing.

**Comment by shgzon** [comment]
If you really stuck and want to continue working on your environment. There is a way to go on. By just removing the entire field and re-adding it again for all sections.

HOWEVER: This results in loss of all data. Big negative..

**Comment by ghost** [comment]
@shgzon Yeah that's not really an option with a couple hundred entries lol.

**Comment by Tam** [comment]
Just pushed some fixes in [v3.5.1](https://github.com/ethercreative/seo/releases/tag/v3.5.1), let me know if it helps! You can manually re-run the upgrade tasks using the `./craft seo/upgrade/to-new-data-format` command.

**Comment by ghost** [comment]
@Tam migration worked! thanks for that. Still having trouble with #168 this after the update though. Do you know anything about this?
